its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ScannerMSBuildTest.java

```diff
@@ -244,6 +244,7 @@ private void assertLineCountForProjectUnderTest(String projectKey) {
 
   @Test
   void testExcludedAndTest_AnalyzeTestProject() throws Exception {
+    assumeTrue(ORCHESTRATOR.getServer().version().isGreaterThanOrEquals(10, 8));
```

Comment:
This seems odd. With the old version, this shouldn't be needed. Do you know why this test failed?

Reply:
No, I can look into it but we might want to merge this one, see the failing CI run [here](https://github.com/SonarSource/sonar-scanner-msbuild/runs/33323871779)

Reply:
You were right and I was wrong, the few dependencies bump indeed had side effects, good catch

Reply:
That's actually good news 👍 . This was really unexpected.